tests/helpers.py

```diff
@@ -14,6 +14,7 @@
 TEST_DATASET_3D_NAME = "[PyTest] Test Dataset 3D"
 TEST_SLICE_NAME = "[PyTest] Test Slice"
 TEST_PROJECT_ID = "60b699d70f139e002dd31bfc"
+# What is this magic?
 TEST_EVAL_FUNCTION_ID = "ef_c61595wh49km7ppkk14g"
```

Comment:
Yeah in fact we don't need this anymore! Now that `list_eval_functions` is good, you can just use the first eval function from there.